src/shipit_static_analysis/shipit_static_analysis/report/mozreview.py

```diff
@@ -178,6 +178,8 @@ def translate_line_num(self, filename, line_num, original=False):
 
         TODO: Convert to a faster search algorithm.
         '''
+        assert isinstance(line_num, int), \
```

Comment:
Nit: the line would be short if you put it everything in one line

Reply:
I find assertions more readable this way...